File: calculatePercentage.java
import java.util.Scanner;

public class calculatePercentage {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter Maths marks: ");
        int maths = sc.nextInt();

        System.out.print("Enter Computer marks: ");
        int computer = sc.nextInt();

        System.out.print("Enter English marks: ");
        int english = sc.nextInt();

        int total = maths + computer + english;
        double percentage = (total / 300.0) * 100;

        System.out.println("Total Marks: " + total);
        System.out.printf("Percentage: %.2f%%", percentage);
    }
}

File: python/calculatePercentage.py
# Calculate total marks and percentage
maths = int(input("Enter Maths marks: "))
computer = int(input("Enter Computer marks: "))
english = int(input("Enter English marks: "))
total = maths + computer + english
percentage = (total / 300.0) * 100
print(f"Total Marks: {total}")
print(f"Percentage: {percentage:.2f}%")
